StepLang.Wiki/TYP005.md

```diff
@@ -0,0 +1,37 @@
+# Variable Already Declared Exception
+
+## Description
+
+This exception is thrown when a variable is declared more than once in the same scope.
+
+Example:
+
+```step
+number a = 1
+
+number a = 2
+```
+
+## Remediation
+
+Make sure that each variable is only declared once in each scope or give each variable a different name.
+
+```diff
+  number a = 1
+- number a = 2
++ number b = 2
+```
+
+## Tip
+
+You can use anonymous code blocks to separate different scopes.
+
+This is valid:
+
+```step
+number a = 1
+
+{
+  number a = 2
+}
+```
```

Comment:
"After the closing brace <kbd>}</kbd>, `a` will be `1` again."

Reply:
Added `println` with comments to the code to show the values